### ISSUE TITLE
Test and finalize fetcher and mapper documentation

### ISSUE THREAD

**Issue by christinklez** [ucldc/rikolti#174]
Test drive documentation, before passing to CIC: Islandora OAI-PMH fetcher/mapper as test case ([row 5 of spreadsheet](https://docs.google.com/spreadsheets/d/1XkBwi8jiuGgrWvQBqjkrbavSl13yyrQNyRyEb_CQ4z0/edit#gid=1733241055))

**Comment by barbarahui** [comment]
Edited Fetcher documentation: https://github.com/ucldc/rikolti/wiki/The-Fetcher-Component/_compare/a2a947abd9c41ace13a8cafcdeb4ae9210994da7...4c4c2c556c19f14f36c86519719e70cbe8f1b267

**Comment by barbarahui** [comment]
Mapper documentation edits: https://github.com/ucldc/rikolti/wiki/The-Mapper-Component/_compare/d4a8f03874a074089c697810d9ae516efaf5c247...41125195d48a0973073e9785f9b162cbc9647b1a

**Comment by barbarahui** [comment]
@amywieliczka we chatted about the jsonl question this morning, but to answer your questions:

- Yes, the mapper output currently produces an array of 100 records
- Yeah, I think it's fine to leave things as 100 records per jsonl file for now. I see what you mean about the tradeoff of optimizing for updating a single record vs file and lambda costs. Make sense to me to wait to see if one kind of optimization outweighs the other.

Also, I updated the wiki page to use your `published` django model manager in the script for finding collections that use a certain mapper type: https://github.com/ucldc/rikolti/wiki/The-Mapper-Component/_compare/41125195d48a0973073e9785f9b162cbc9647b1a...2309812853fe192dc69e3951fe164d53d544ea6d
